### PR TITLE
Add BaseHelper test

### PR DESCRIPTION
**Pull request by ekigamba**

### CODE REVIEW

File: opensrp-app/build.gradle [hunk 1]
```diff
@@ -128,11 +125,26 @@ android {
...
+                TestLogEvent.SKIPPED,
+                TestLogEvent.STANDARD_OUT
+        exceptionFormat TestExceptionFormat.FULL
+        showExceptions true
+        showCauses true
+        showStackTraces true
+
+        error {
+            exceptionFormat = 'full'
+        }
```
Comment (ellykits) [inline opensrp-app/build.gradle:147]: Is this needed for GitHub actions now that we switched from travis?

**Review by ellykits — COMMENTED** [review]
Left a comment

File: opensrp-app/build.gradle (hunk 1 above)
Comment (ekigamba) [reply opensrp-app/build.gradle:147]: Yes, I added this for Github actions specifically. I don't think there is an alternate option to enable the required logging

**Comment by ellykits** [comment]
:+1: Looks good.

**Review by ellykits — APPROVED** [review]

**Review by ellykits — APPROVED** [review]

**Pull request merged**